index.md

```diff
@@ -277,100 +277,84 @@ href="{{site.swc_site}}">Software Carpentry</a>, <a href="{{site.dc_site}}">Data
   <a href="https://goo.gl/forms/KoUfO53Za3apOuOK2">
-    <button type="button" class="btn btn-info">Report a Code of Conduct Incident</button>
+    <button type="button" class="btn btn-info">Повідомити про інцидент, пов'язаний із кодексом поведінки</button>
   </a>
 </p>
 <hr/>
 
 
 {% comment %}
-Collaborative Notes
+Нотатки для співпраці
 
-If you want to use an Etherpad, go to
+Якщо ви хочете використовувати Etherpad, перейдіть до
 
 https://pad.carpentries.org/YYYY-MM-DD-site
 
-where 'YYYY-MM-DD-site' is the identifier for your workshop,
-e.g., '2015-06-10-esu'.
+де 'YYYY-MM-DD-site' є ідентифікатором вашого семінару,
+Наприклад, '2015-06-10-esu'.
 
-Note we also have a CodiMD (the open-source version of HackMD)
-available at https://codimd.carpentries.org
+Зверніть увагу, що у нас також є CodiMD (версія HackMD з відкритим вихідним кодом), доступна за адресою https://codimd.carpentries.org
 {% endcomment %}
 {% if page.collaborative_notes %}
-<h2 id="collaborative_notes">Collaborative Notes</h2>
+<h2 id="collaborative_notes">Нотатки про співпрацю</h2>
 
 <p>
-We will use this <a href="{{ page.collaborative_notes }}">collaborative document</a> for chatting, taking notes, and sharing URLs and bits of code.
+Ми використовуватимемо цей <a href="{{ page.collaborative_notes }}">спільний документ</a> для спілкування в чаті, створення нотаток і обміну URL-адресами та фрагментами коду.
 </p>
 <hr/>
 {% endif %}
 
 
 {% comment %}
-SURVEYS - DO NOT EDIT SURVEY LINKS
+ОПИТУВАННЯ - НЕ РЕДАГУЙТЕ ПОСИЛАННЯ НА ОПИТУВАННЯ
 {% endcomment %}
 <h2 id="surveys">Surveys</h2>
-<p>Please be sure to complete these surveys before and after the workshop.</p>
+<p>Будь ласка, обов'язково заповніть ці опитування до та після семінару.</p>
 {% if site.carpentry == "incubator" %}
-<p><a href="{{ site.incubator_pre_survey }}">Pre-workshop Survey</a></p>
-<p><a href="{{ site.incubator_post_survey }}">Post-workshop Survey</a></p>
+<p><a href="{{ site.incubator_pre_survey }}">Опитування перед семінаром</a></p>
+<p><a href="{{ site.incubator_post_survey }}">Опитування після семінару</a></p>
 {% elsif site.incubator_pre_survey or site.incubator_post_survey %}
 <div class="alert alert-danger">
-WARNING: you have defined custom pre- and/or post-survey links for
-a workshop not configured for The Carpentries Incubator
-(the value of `curriculum` is not set to `incubator` in `_config.yml`).
-Please comment out the `incubator_pre_survey` and `incubator_post_survey` fields
-in `_config.yml` or, if this workshop is teaching a lesson in the Incubator,
-change the value of `carpentry` to `incubator`.
+УВАГА: ви визначили власні посилання до та/або після опитування для семінару, який не налаштований для The Carpentries Incubator (значення 'curriculum' не встановлено на 'incubator' у '_config.yml'). Будь ласка, закоментуйте поля «incubator_pre_survey» та «incubator_post_survey» у «_config.yml» або, якщо цей семінар проводить урок в Incubator, змініть значення «carpentry» на incubator.
```

Comment:
@VeronikaShevc please use same kinds of quotes, don't replace them by "fancy quotes" like «carpentry»